rfd/0156-notifications.md

```diff
@@ -131,24 +149,30 @@ These will be stored under `/notification/global/<notification id>`.
 ```protobuf
 // GlobalNotification represents a global notification.
 message GlobalNotification {
+   // kind is the resource kind ("user_notification_state").
```

Comment:
Copy paste error? (notification state?)

Do we actually need these in the protos? If `kind` can never be anything other than one specific value, can we omit it from the proto and just have a custom marshaler that hard codes it? @rosstimothy is this a bad idea?